kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -0,0 +1,1180 @@
+                    raise Exception('failed to delete bridge %s: %s' % (br_name, e))
+                logger.info('deleted managed bridge %s' % br_name)
+
+        # 4. Stop openvswitch (includes ovs-vswitchd, and ovsdb-server on
+        #    distros that bundle them into a single service unit)
+        r, o, e = bash.bash_roe('systemctl stop openvswitch')
+        if r != 0:
+            raise Exception('failed to stop openvswitch: %s' % e)
+        logger.info('stopped openvswitch')
+
+        # 5. Stop ovsdb-server if it is a separate service unit
+        r, o, e = bash.bash_roe('systemctl stop ovsdb-server')
```

Ruff (0.15.6)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)